### DISCUSSION TITLE
Emoji Display Issue on GitHub Mobile (Android-version)

### DISCUSSION THREAD

**Discussion by peymansf2000** [community/community#101795] [Mobile] [answered]
### Select Topic Area

Bug

### Body

I've noticed an issue on GitHub for Android where some emojis are not displaying properly. This problem might also be affecting iOS(I haven't tested it).
I've attached screenshots of the emoji display from both the browser and the Android version for better understanding.

Github mobile android version

![IMG_20240124_124548](https://github.com/community/community/assets/20140256/df6f0be6-055f-489c-b936-71cfc45fa7af)

Github in browser

![IMG_20240124_130113](https://github.com/community/community/assets/20140256/b01af399-b9df-4969-a14b-3c5c7b79a387)

[link of discussion ](https://github.com/ParanoidUser/yolo/discussions/categories/galaxy-trivia?discussions_q=category%3A%22Galaxy+Trivia%22+is%3Aclosed)

**Comment by eliperkins** [comment, marked as answer]
Hey @peymansf2000, thanks for the heads up on this one! The images you shared here are quite helpful, so thanks for those. This is definitely a bug and we can polish this up.

I've filed an issue on our internal tracking repo for this bug. I'll work with our engineering team to get it addressed and follow back up when the fix goes live. Thanks for helping us improve GitHub Mobile!
